packages/node/test/integration/setup.ts

```diff
@@ -26,6 +54,37 @@ export async function setup(
   const messagingService = firebaseServiceFactory.createMessagingService(
     process.env.FIREBASE_MESSAGING_SERVER_KEY!
   );
+
+  const postgresServiceFactory = new PostgresServiceFactory({
```

Comment:
As evidenced here. Why do we have a "PostgresServiceFactory"? We just need a "PostgresStoreService".

Reply:
This was my fault, I tried to abide by the given interfaces as much as possible, the factory did seem unnecessary to me then as well. I will remove this.

Reply:
Or it looks like you still might be deep into this @cf19drofxots, let me know if you want me to do any of this.

Reply:
As said in the other comment, this gives more flexibility in creating services, as compared to creating multiple services separately which would amount to effectively the same thing as having a factory producing the services.